<?xml version="1.0" encoding="utf-8"?>

<launch>

    <include file="$(find elevation_mapping_cupy)/launch/turtlesim_init.launch">
        <arg name="rviz_config" value="$(find elevation_mapping_cupy)/rviz/turtle_example.rviz"/>
    </include>


    <node pkg="semantic_sensor" type="image_node.py" name="front_cam" args="front_cam"
          output="screen">
        <rosparam command="load" file="$(find elevation_mapping_cupy)/config/image_semantics.yaml"/>
    </node>

        <node pkg="elevation_mapping_cupy" type="elevation_mapping_node" name="elevation_mapping" output="screen">
            <rosparam command="load" file="$(find elevation_mapping_cupy)/config/semantic_parameters.yaml"/>
            <rosparam command="load" file="$(find elevation_mapping_cupy)/config/image_semantic_sensor_parameter.yaml"/>
        </node>
</launch>